_includes/git-info.html

```diff
@@ -1,3 +1,5 @@
+{% if site.guardian.style.git_info_enabled %}
```

Comment:
Nit This could just use the existence or non-existence of `github_repo` to determine if this stuff should show.

Reply:
good call a829f27